### PR TITLE
added an example to help clarify relations

### LINKED ISSUE #1669: details on relation-{list,ids,get} are lacking.
I've found this to be useful: https://gist.github.com/jrwren/d19672314703010d60dec10f604cf185

Could someone help me modify it for inclusion in the docs?

### PR DESCRIPTION
**Pull request by matthewhelmke**
Fixes #1669 

@jrwren , thanks again for the example and information! Please review my changes to make sure I did not accidentally introduce errors in the understanding.

@juju/docs, please do a typical review after jrwren's is complete.

### CODE REVIEW

File: src/en/authors-hook-environment.md [hunk 1]
```diff
@@ -472,18 +495,67 @@ server:7
...
+```
+
+Which will return the unit(s) for the given ID, like this:
+
+```bash
+kibana/2
+```
+
+In this example, only one unit was returned in the output, `kibana/2` because
+the kibana application only has one unit. If more than one unit existed, all
```
Comment (evilnick) [inline src/en/authors-hook-environment.md]: Kibana application

File: src/en/authors-hook-environment.md [hunk 2]
```diff
@@ -472,18 +495,67 @@ server:7
...
+would be listed in the output.
+
+You can use this output with `relation-get` to find the relation data for each
+unit, like this:
+
+```bash
+juju run --unit haproxy/0 'relation-get -r reverseproxy:115 - kibana/2'
+```
+
+This returns the data that the kibana charm in our example is sending to the
```
Comment (evilnick) [inline src/en/authors-hook-environment.md]: Kibana

Comment (matthewhelmke) [inline src/en/authors-hook-environment.md]: Thanks!

File: src/en/authors-hook-environment.md (hunk 1 above)
Comment (matthewhelmke) [inline src/en/authors-hook-environment.md]: Thanks!

**Pull request merged**